feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/presentation/viewmodel/FavouritesState.kt

```diff
@@ -0,0 +1,10 @@
+package com.epmedu.animeal.favourites.presentation.viewmodel
+
+import com.epmedu.animeal.feeding.domain.model.FeedingPoint
+
+data class FavouritesState(
+    val favouritesSnapshot: List<FeedingPoint> = emptyList(),
```

Comment:
Looks like you don't need this list on UI. Move it to the viewmodel as a private variable instead